ui/app/AppLayouts/Chat/panels/communities/CommunityEditSettingsPanel.qml

```diff
@@ -93,8 +93,8 @@ StatusScrollView {
 
                 StatusStackModal {
                     width: 640
-                    height: root.height
```

Comment:
This breaks the color picker.

<img width="1312" alt="Screenshot 2023-04-04 at 08 37 07" src="https://user-images.githubusercontent.com/47811206/229697808-21e13bc8-d0fc-4a7d-820a-a573c8d160ad.png">

Reply:
Ye, the `e2e` tests are righfully failing :) I'll look into this

Reply:
Ah I know what's wrong, it needs the other fix from https://github.com/status-im/status-desktop/pull/10145; it worked for me because I had them initially all together :D

Reply:
Just rebased and all good now:
![image](https://user-images.githubusercontent.com/5377645/229752362-1edd5f0f-848d-4d8e-9a59-24b7f92ad842.png)

Reply:
@alexjba and the `e2e` test suite is green again :)